src/media/js/content_filter.js

```diff
@@ -43,7 +47,7 @@ define('content_filter',
             return {};
         }
         return {
-            doc_type: filterContent
+            doc_type: settings.meowEnabled ? filterContent : 'webapp'
```

Comment:
I would have done that above with the rest of the `filterContent` checks, but that's just me.